### ISSUE TITLE
define prerequisites

### ISSUE THREAD

**Issue by CFenner** [RobertShippey/MMM-PlexDeck#1]
What are the prerequisites to run this module? My MM is showing `unauthorized` and I don't know what to do.

**Comment by RobertShippey** [comment]
Hi @CFenner - thanks for getting in touch!

I'm a bit of a MagicMirror noob so sorry if the documentation isn't clear enough yet. Also, I've only tested this on my version of Plex so hopefully their undocumented API hasn't/doesn't change much. 

It sounds like you're using the `.plex.direct` domain in the config.js file? Unfortunately the API isn't exposed through this proxy service. You should use the actual IP of the Plex Media Server on your local network (or a local name that resolves to a local IP). You can find the right IP and port in Settings > Remote Access > Show Advanced, right next to Private.  

If your mirror and Plex server are on different networks then you'll probably need to reconfigure your router or something, which is out of the scope of this module. 

Let me know if this fixes the issue!

**Comment by zeular** [comment]
Hi
I'm not getting this module to work, nothing happens, it's just black. I had set my IP and port. 
Actually, I would like to have the "Last movie added" instead of "On Deck". Is that possible?

**Comment by RobertShippey** [comment]
@zeular Could you let me know which version of Plex Media server you're using and I'll do a little investigation.

**Comment by CriticalPoint** [comment]
I'm was trying to access this via DNS resolution, but that didn't work, so I have set it to my Plex public IP and I'm still seeing the same, which is a black module. I'll bring this in-house, so to speak and test with a local 192, which should at least rule out the network which problem solving.

Does your module support the 'position' configuration option? Specifically 'lower_third'?

**Comment by zeular** [comment]
@RobertShippey Yes, here it is: Version 1.5.5.3634, I have added the ip of the MM to plex "alloed list".

**Comment by CFenner** [comment]
@CriticalPoint with `http://<ip>:32400/library/onDeck` I get an "unauthorized". Do I need to activate it?
I'm using `Version 1.5.3.3580`

**Comment by RobertShippey** [comment]
It turns out that I'm using a really out-of-date version of Plex and I had no idea. I'm having the same issues you are while testing with the latest version. 

I'll continue to investigate and get back to you.

**Comment by anderssonoscar0** [comment]
Any progress? ✌️

**Comment by bran** [comment]
This module has many naming problems and tries to parse an XML (from Plex) that is very outdated. See #2 for working fix.
